app/src/main/java/ch/epfl/sdp/peakar/user/challenge/goal/RemotePointsChallenge.java

```diff
@@ -45,7 +54,7 @@ public RemotePointsChallenge(String id, List<String> users, long awardPoints, in
      * @param durationInDays duration of the challenge in number of days
      */
     @SuppressLint("NewApi")
-    public static Challenge generateNewChallenge(String founderID, int durationInDays) {
+    public static Challenge generateNewChallenge(String founderID, String challengeName, int durationInDays) {
```

Comment:
The new parameter is missing in the doc.

Reply:
resolved